home_work_11/index.js

```diff
@@ -0,0 +1,108 @@
+import { mockDataArray } from "./mockData.js";
+
+const root = document.createElement("div");
+
+const containerValueOfProducts = document.createElement("div");
+containerValueOfProducts.classList.add("container");
+
+const productList = document.createElement("ul");
+productList.classList.add("product-list");
+
+root.prepend(productList, containerValueOfProducts);
+document.body.prepend(root);
+
+function showDetails(productName) {
+  const productDetailsBlock = document.getElementById(productName);
+  if (productDetailsBlock.style.display === "none") {
+    productDetailsBlock.style.display = "block";
+    return;
+  } else {
+    productDetailsBlock.style.display = "none";
+    return;
+  }
+}
+
+function renderProductsListItem(
+  productName,
+  productPrice,
+  certificate,
+  productCountry,
+  weight,
+  dateOfExpiry,
+  sugarFree,
+  productProvider
+) {
+  return `<div class = "product-item">
+    <span class = "certificate-attation">${certificate ? "" : "!"}</span>
+    <span class = "product-name">${productName}</span>
+    <span class = "product-price">: for ${weight} grams, the price ${productPrice} UAH</span>
+    <img style = "width:30px; margin-left: 6px" src = "${productCountry}" alt = "image-flag">
```

Comment:
Try to no add inline styles via JS into HTML. It can lead to specificity issues. Toggle classes instead or just simply add a CSS rule, like so:

```
.product-item img {
    width:30px; 
    margin-left: 6px
}
```